rules/EarlyReturn/Rector/If_/ChangeAndIfToEarlyReturnRector.php

```diff
@@ -238,7 +238,7 @@ private function shouldSkip(If_ $if, ?Stmt $nexStmt): bool
             return true;
         }
 
-        // is simple return? skip it
+        // is simple ifNextReturn? skip it
```

Comment:
It seems search replace typo

Reply:
Reporting once is enough :D I'll just search and replace all at once